src/common/types.cpp

```diff
@@ -1656,7 +1656,7 @@ idx_t StructType::GetChildCount(const LogicalType &type) {
 bool StructType::IsUnnamed(const LogicalType &type) {
 	auto &child_types = StructType::GetChildTypes(type);
 	if (child_types.empty()) {
-		return false;
+		return true;
```

Comment:
This was necessary to fix a problem with:
```sql
from 'data/parquet-testing/variant_object_empty.parquet';
```

Though it did change the result from `{}` to `()`, because unnamed structs are rendered as tuples: `(` <content> `)`